### DISCUSSION TITLE
Approved, but no benefits

### DISCUSSION THREAD

**Discussion by darkpsy3934** [community/community#61099] [GitHub Education]
### Hi everyone,

I was approved on the 15th of July and saw I'd have access to the benefits within the hour.  It's now two days later and I do not have an email nor can I access the benefits.

Can anyone please advise?

Thanks!

**Comment by the-officialjosh** [comment]
Wait it was clear in your acceptance prompt. It would be rolled out to you

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
